src/features/user/ProfileV2/components/MicroComponents/Project.tsx

```diff
@@ -0,0 +1,99 @@
+import TreeIcon from '../../../../../../public/assets/images/icons/TreeIcon';
+import getImageUrl from '../../../../../utils/getImageURL';
+import { useTranslation } from 'next-i18next';
+import { getDonationUrl } from '../../../../../utils/getDonationUrl';
+import formatDate from '../../../../../utils/countryCurrency/getFormattedDate';
+
+const Project = ({ projectInfo }: ProjectProps): ReactElement => {
+  const { token } = useUserProps();
+  const { embed } = useContext(ParamsContext);
+  const { t } = useTranslation(['me', 'country']);
+  const handleDonate = (slug: string) => {
+    const url = getDonationUrl(slug, token);
+    embed === 'true' ? window.open(url, '_top') : (window.location.href = url);
+  };
+  return (
+    projectInfo && (
+      <div className={myForestStyles.donationDetail}>
+        <div className={myForestStyles.image}>
+          {projectInfo?.plantProject !== null ? (
+            <img
+              src={getImageUrl(
+                'project',
+                'medium',
+                projectInfo.plantProject.image
+              )}
+              width="100%"
+              height="100%"
+            />
+          ) : (
+            <div className={myForestStyles.registerTreeIcon}>
+              {projectInfo?.treeCount && projectInfo?.treeCount > 1 ? (
+                <TreesIcon />
+              ) : (
+                <TreeIcon />
+              )}
+            </div>
+          )}
+        </div>
+        <div className={myForestStyles.projectDetailContainer}>
+          <div className={myForestStyles.projectDetail}>
+            <div style={{ maxWidth: '345px' }}>
+              <p className={myForestStyles.projectName}>
+                {projectInfo?.plantProject !== null
+                  ? projectInfo.plantProject.name
+                  : 'Registered Trees'}
+              </p>
+              {projectInfo?.plantProject !== null && (
+                <div>
+                  {t(
+                    'country:' + projectInfo.plantProject.country.toLowerCase()
+                  )}
+                  -{projectInfo.plantProject.tpo.name}
+                </div>
+              )}
+            </div>
+            <div className={myForestStyles.treeCount}>
+              {console.log(projectInfo.quantity, projectInfo?.treeCount, '===')}
+              {projectInfo?.plantProject?.unit === 'm2'
+                ? t('me:area', { areaConserved: `${projectInfo.quantity}` })
+                : t(
+                    (projectInfo?.quantity !== null &&
+                      projectInfo?.quantity > 1) ||
+                      (projectInfo?.treeCount !== null &&
+                        projectInfo?.treeCount > 1)
+                      ? 'plantedTrees_other'
+                      : 'me:plantedTrees_one',
+                    {
+                      count:
+                        projectInfo.quantity || projectInfo?.treeCount || 0,
+                    }
+                  )}
```

Comment:
Sorry, but why have you changed this to manually determine the right resource for singular/plural and do not let the I18N module do it automatically as before?